.github/workflows/_tauri.yml

```diff
@@ -33,7 +33,7 @@ jobs:
             syms-artifact: rust/gui-client/firezone-client-gui-linux_1.1.2_x86_64.dwp
             # mark:next-gui-version
             pkg-artifact: rust/gui-client/firezone-client-gui-linux_1.1.2_x86_64.deb
-          - runs-on: windows-2019
+          - runs-on: windows-2019-firezone
```

Comment:
These appear to benefit by ~2minutes, probably because they are release builds and thus compiling / linking them is more CPU bound?

Reply:
Yeah the release builds get really intense LTO to reduce binary size. (In another thread I thought it was for performance optimizations)
https://github.com/firezone/firezone/blob/184c982d756076734ae62ae4ec463b91b0e4bec8/rust/Cargo.toml#L71-L79

I forgot which thread, but I agree that we could just build with LTO for `main` and then build unoptimized MSIs and debs for PRs, since we can build optimized packages locally if we need anyway.

Reply:
We can set these parameters via env variables in CI, then it is much easier to control depending on the branch.

I'll see if I can get that done quickly.

Reply:
Just FYI I'm not sure if these larger runners are worth the cost? Looks like they'll add up to a sizable bill even with credits :-/

Can do some final testing to be sure

Reply:
I'd be in favor of not building proper release binaries in each PR. Chances that those are used pretty smallz right? Esp. when the optimisations are just around binary size and not speed.